plugins/topology/src/components/Graph/decorators/PipelineRunDecorator.tsx

```diff
@@ -6,9 +6,9 @@ import {
   getLatestPipelineRun,
   getTaskRunsForPipelineRun,
   pipelineRunStatus,
+  Status,
```

Comment:
In PipelineRunDecorator, success icon (check mark) should be green in colour.

![image](https://github.com/user-attachments/assets/af9cc3fc-2b92-4785-9a6b-2decf5ad18d8)

Reply:
Looks like there isn't any straight forward way to override PF styles here, I did wrap the Status Icon component with  overriding SVG fill property to retain the theme colors. See if this approach works for you https://gist.github.com/karthikjeeyar/c584caa00d3da30b06ab41269b726a01

Reply:
Fixed. Thank you for the suggestion
![Screenshot 2024-09-18 at 2 33 56 PM](https://github.com/user-attachments/assets/c6375806-30a2-40b6-987b-b24a5d572a86)